mathutil_test.go

```diff
@@ -0,0 +1,107 @@
+package mathutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
I would prefer to not use any testing library except stdlib's `testing` package.

Not critical, can be resolved later.

Reply:
sure, I'll remove the dependency when I move the tests to `all_test.go`